packages/@webex/plugin-meetings/src/breakouts/index.ts

```diff
@@ -104,8 +104,12 @@ const Breakouts = WebexPlugin.extend({
       leading: true,
       trailing: false,
     });
-    this.listenTo(this.breakouts, 'add', () => {
+    this.listenTo(this.breakouts, 'add', (breakout) => {
       this.debouncedQueryRosters();
+      this.triggerReturnToMainEvent(breakout);
```

Comment:
need to check the requested attribute when breakout add, because if breakout add with the requested attribute, the `change` event will not trigger.